.eslintrc

```diff
@@ -9,6 +9,8 @@ ecmaFeatures:
   binaryLiterals: true
   generators: true
   forOf: true
+  objectLiteralShorthandMethods: true
+  objectLiteralShorthandProperties: true
```

Comment:
+1, we had to enable one of these in https://github.com/nodejs/io.js/pull/1760 already.